styles.css

```diff
@@ -302,3 +306,102 @@ body {
   margin-left: 16px;
   font-size: 1em;
 }
+
+.contact {
+  border-top-left-radius: 105px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 30px;
+  background: #6070ff url(images/contact-form-shapes-mobile.png) no-repeat center;
+  background-size: 100% 90%;
+  margin-top: 25%;
+}
+
+.contact-me {
+  font-family: 'Poppins', sans-serif;
+  width: 100%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.2em;
+  line-height: 52px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  margin-top: 50%;
+}
+
+.contact-info {
+  width: 100%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 28px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: #ebebff;
+  margin: 0 auto;
+}
+
+.form button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
- [ ] Great job on implementing the contact form of your portfolio as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address and fix before moving on to the next milestone. To do so kindly consider the below suggestion:
   - [ ] Kindly consider adding a `hover` effect to your **form button** to match that of the `Figma` design. It would enhance the user experience and provide visual feedback. You can achieve this by utilizing CSS and applying a `hover` pseudo-class selector to the `button` element. Consider adding appropriate CSS properties such as background color and text color to create the desired hover effect. See below for more details: 
   
<img width="900" alt="Screenshot 2023-07-11 at 19 18 52" src="https://github.com/yayaismaail/my-portfolio/assets/107192076/8212e479-ab65-4e8e-b46d-555dec7e504d">